test/test_rack_server.rb

```diff
@@ -20,21 +20,22 @@ def call(env)
         @app.call(env)
       rescue Exception => e
         @exception = e
+STDOUT.syswrite "\n#{e.message}\n"
         [ 500, {}, ["Error detected"] ]
       end
     end
   end
 
   class ServerLint < Rack::Lint
     def call(env)
-      check_env env
+      check_env env if Rack::RELEASE < '3'
```

Comment:
Didn't see removal of this method mentioned in the changelog for rack 3 https://github.com/rack/rack/blob/main/CHANGELOG.md

fyi @ioquatix

Reply:
Looks like it was renamed to `check_environment` in https://github.com/rack/rack/commit/eef251d3758741fec514356c117ff791a4c23347

Reply:
Actually, `Wrapper.new(@app, env).check_environment`.  Added.